File: CS/CustomParameterEditorsWPF/Categories.cs
using System.Collections.Generic;
using CustomParameterEditorsWPF.App_data;
// ...

namespace CustomParameterEditorsWPF {
    public class Categories : List<string> {
        public Categories() {
            nwindDataSet dataSet = new nwindDataSet();
            new CustomParameterEditorsWPF.App_data.nwindDataSetTableAdapters.CategoriesTableAdapter().Fill(dataSet.Categories);
            foreach(var category in dataSet.Categories) {
                Add(category.CategoryName);
            }
        }
    }
}

File: VB/CustomParameterEditorsWPF/Categories.vb
Imports System.Collections.Generic
Imports CustomParameterEditorsWPF.App_data

' ...
Namespace CustomParameterEditorsWPF

    Public Class Categories
        Inherits List(Of String)

        Public Sub New()
            Dim dataSet As nwindDataSet = New nwindDataSet()
            Call New nwindDataSetTableAdapters.CategoriesTableAdapter().Fill(dataSet.Categories)
            For Each category In dataSet.Categories
                Add(category.CategoryName)
            Next
        End Sub
    End Class
End Namespace
